src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -685,6 +686,8 @@ public static String getInstructorHoverMessageForFeedbackSession(FeedbackSession
     
     /**
      * Returns the links of actions available for a specific session
+     * @param hasActions
+     *         Indicates whether or not there are possible actions in the first place
```

Comment:
Are there any sessions that has no actions? Or is this for like some kind of error situation? Could you point me in the right direction for this?